website/content/commands/acl/auth-method/create.mdx

```diff
@@ -7,8 +7,18 @@ page_title: 'Commands: ACL Auth Method Create'
 
 Command: `consul acl auth-method create`
 
+Corresponding HTTP API Endpoint: [\[PUT\] /v1/acl/auth-method](https://www.consul.io/api-docs/acl/auth-methods#create-an-auth-method)
```

Comment:
I realize this will be a somewhat annoying fix (because it applies to every command), but these links should be modified to remove the `www.consul.io` (so they reference their own deployment rather than the current production site).

```suggestion
Corresponding HTTP API Endpoint: [\[PUT\] /v1/acl/auth-method](/api-docs/acl/auth-methods#create-an-auth-method)
```

Reply:
All links updated to relative paths.